pom.xml

```diff
@@ -283,6 +283,18 @@
         <version>${powermock.version}</version>
         <scope>test</scope>
       </dependency>
+
```

Comment:
I think top-level is fine as long as the scope keeps the dependency out of the jar.